test/plugin.js

```diff
@@ -207,6 +218,60 @@ describe("nemo-drivex @plugin@", function () {
     });
   });
   /**
+   * waitForElementNotVisible Wait for timeout milliseconds for the WebElement to be NOT visible
+   * @param locator {LocatorJSON}
+   * @param timeout {Number}
+   * @param msg {String} optional message for any error messages
+   * @returns {Promise} resolves to true or false
+   */
+  describe("@waitForElementNotVisible@ method", function () {
+    it("should not be visible", function () {
+      assert.ok(nemo.drivex.waitForElementNotVisible);
+    });
+    it("should return true when element is not visible", function (done) {
+      nemo.driver.get('https://warm-river-3624.herokuapp.com/waits');
+      util.waitForJSReady(nemo);
+      nemo.drivex.waitForElementNotVisible({
+        'locator': 'outy',
+        'type': 'id'
+      }, 6000, "found outy div visible").then(function (notVisible) {
+        assert.equal(notVisible, true);
+        done();
+      }, util.doneSuccess(done));
+    });
+    it("should reject promise when element is visible", function (done) {
+      nemo.driver.get('https://warm-river-3624.herokuapp.com/waits');
+      util.waitForJSReady(nemo);
+      nemo.drivex.find({locator: '#wrapper > form > input', type: 'css'}).click();
+      nemo.driver.sleep(5000); // Waiting for element to be visible
+      nemo.drivex.waitForElementNotVisible({
+        'locator': 'outy',
+        'type': 'id'
+      }, 6000, "found outy tag").then(function (notVisible) {
+        assert.equal(notVisible, false);
```

Comment:
I think you want to call `done` with an error here, as you expect the promise to be rejected and thus this function will ONLY execute if the `waitForElementVisible` function has some defect.